runtime/common/src/paras_registrar.rs

```diff
@@ -1059,6 +1101,124 @@ mod tests {
 			assert_noop!(Registrar::swap(Origin::signed(1), para_id, para_id + 2), BadOrigin);
 		});
 	}
+
+	#[test]
+	fn swap_handles_bad_states() {
+		new_test_ext().execute_with(|| {
+			let para_1 = LOWEST_PUBLIC_ID;
+			let para_2 = LOWEST_PUBLIC_ID + 1;
+			run_to_block(1);
+			// paras are not yet registered
+			assert!(!Parachains::is_parathread(para_1));
+			assert!(!Parachains::is_parathread(para_2));
+
+			println!("{:?}", paras::Pallet::<Test>::lifecycle(para_1));
+			println!("{:?}", paras::Pallet::<Test>::lifecycle(para_2));
```

Comment:
Did you mean to keep these `prinln!`s in?

Reply:
probably not